minio/minio
# Checking for access denied in web browser request.

Fixes #9485

## Description


## Motivation and Context


## How to test this PR?
One possible/easy way to test: Create/Put/Copy objects to a bucket that can be locked and is locked. Try to delete the object(s) copied in browser. Instead of seeing 'We encountered an internal error. Please try again.' you should see a more appropriate message.

## Types of changes

## Checklist:
